### ISSUE TITLE
Дублирование фото

### ISSUE THREAD

**Issue by BishopGIS** [nextgis/tree_mapper#21]
При отправке фото деревьев происходит дублирование:
http://citywoody.nextgis.com/resource/10/feature/529

**Comment by 4eRTuk** [comment]
Тоже не могу поймать. Возможно, было из-за слабого интернета или битого кеша. Метод используется тот же самый что и в NGM.

**Comment by BishopGIS** [comment]
Да, думаю тут именно в интернете дело. Но тут на подумать - как это обходить. 
Кейс видимо вот такой:

1. Мобильное отрпавляет фотку
2. Фотка закачивается на сервер и сервер посылает ответ 200
3. Мобильное не получает код и пробует закачать заново - тут вроде должен был сработать механизм проверки изменений на сервере до отправки своих правок - и видимо не сработал.

**Comment by 4eRTuk** [comment]
Механизм проверки изменений то работает, вот только после отправки на сервер как раз таки мобильное ожидает 200 код и не получает его, соответственно локальные правки остаются и ид записи не меняется на новый с сервера.
